test/common/router/scoped_rds_test.cc

```diff
@@ -237,6 +237,8 @@ route_configuration_name: foo_routes
 )EOF";
   Protobuf::RepeatedPtrField<ProtobufWkt::Any> resources;
   parseScopedRouteConfigurationFromYaml(*resources.Add(), config_yaml);
+  init_watcher_.expectReady().Times(1);
```

Comment:
Nit: `Times(1)` is redundant, that is the default.

Reply:
done